fluentd/lib/fluent-plugin-prometheus/in_prometheus_tail_monitor.rb

```diff
@@ -51,11 +51,11 @@ def start
         inode: get_gauge(
           :fluentd_tail_file_inode,
           'Current inode of file.'),
-        totalbytes_logged_in_disk: get_gauge(
-          :fluentd_totalbytes_logged_in_disk,
+        totalbytes_logged: get_gauge(
+          :fluentd_totalbytes_logged,
           'totalbytes logged in disk.'),
-        totalbytes_collected_from_disk: get_gauge(
-          :fluentd_totalbytes_collected_from_disk,
+        totalbytes_collected: get_gauge(
```

Comment:
Consistent underscores: total_bytes_collected
The metrics should be counters, not gauges. The total bytes collected always increases. When fluentd restarts the counter will reset to 0. Prometheus will handle this correctly for a counter metric (continue to accumulate the total) but for a guage metric the value at prometheus will drop to 0. Guages are intended for values that can go up and down.